b.go

```diff
@@ -0,0 +1,4 @@
+// hoge
+package a
+
+var Y = 2
```

Comment:
__[Golint]__

exported var Y should have comment or be unexported


<!-- Comment by [SideCI](https://sideci.com) -->